media-video/mpv/mpv-0.25.0-r1.ebuild

```diff
@@ -45,7 +45,7 @@ REQUIRED_USE="
 	gbm? ( drm egl opengl )
 	lcms? ( opengl )
 	luajit? ( lua )
-	opengl? ( || ( aqua egl X raspberry-pi !cli? ( libmpv ) ) )
+	opengl? ( || ( aqua egl X raspberry-pi !cli ) )
```

Comment:
You are mistaken. Currently it reads 'opengl is satisfied by libmpv without cli, but not with cli'. Otherwise one can enable opengl, cli, libmpv, but disable aqua, egl, X, raspberry-pi and have no working video.